glib/src/auto/functions.rs

```diff
@@ -54,6 +54,13 @@ pub fn base64_encode(data: &[u8]) -> crate::GString {
 //    unsafe { TODO: call ffi:g_base64_encode_step() }
 //}
 
+#[doc(alias = "g_blow_chunks")]
+pub fn blow_chunks() {
+    unsafe {
+        ffi::g_blow_chunks();
+    }
+}
```

Comment:
a bunch of functions were added here, needs careful checking

Reply:
Opened https://gitlab.gnome.org/GNOME/glib/-/issues/3231 for some of those questions

Reply:
Removed it, see last commit